app/controllers/draft_events_controller.rb

```diff
@@ -58,69 +59,109 @@ def next_player
+        round_number: controller_store[:round_number],
+      }
 
+      broadcast_message :join_draft, _message, namespace: :drafts
+
+      if controller_store[:players].length == controller_store[:max_players]
+        WebsocketRails.users[1].send_message :ready_to_start, {},
+                                             namespace: :drafts
+      end
+    else
       _message = {
-        pick: pick,
-        contestant_name: pick.contestant.name,
-        contestant_id: message["contestant_id"],
-        success: true
+        success: false,
+        message: "Draft Already Started"
       }
+      WebsocketRails.users[message["user_id"]].send_message :join_draft,
+                                                            _message,
+                                                            namespace: :drafts
+    end
+  end
 
-      if pick.save
-        WebsocketRails.users[message["user_id"]].send_message :take_player,
-                                                              _message,
-                                                              namespace: :drafts
+  def pick_contestant
+    draft = Draft.find message["draft_id"]
+    if draft.started
+      if active_player == current_user
+        pick = Pick.new
+        pick.user_id = active_player.id
+        pick.draft_id = message["draft_id"]
+        pick.contestant_id = message["contestant_id"]
+        pick.pick_type = message["type"]
+
+        _message = {
+          pick: pick,
+          contestant_name: pick.contestant.name,
+          contestant_id: message["contestant_id"],
+          success: true
+        }
+
+        if pick.save
+          WebsocketRails.users[message["user_id"]].send_message :take_player,
+                                                                _message,
+                                                                namespace: :drafts
 
-        broadcast_message :pick_contestant, _message, namespace: :drafts
-        next_player
+          broadcast_message :pick_contestant, _message, namespace: :drafts
+          next_player
+        end
+      else
+        # broadcast message to emitter that they are not the active player
+        WebsocketRails.users[message["user_id"]].
+            send_message :pick_contestant,
+                         {
+                           success: false,
+                           message: "You are not the active player. " +
+                               "Please wait your turn."
+                         },
+                         namespace: :drafts
       end
     else
       # broadcast message to emitter that they are not the active player
       WebsocketRails.users[message["user_id"]].
           send_message :pick_contestant,
                        {
-                         success: false,
-                         message: "You are not the active player. " +
-                             "Please wait your turn."
+                           success: false,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.